client/src/main/java/org/jboss/fuse/mvnd/client/Installer.java

```diff
@@ -105,15 +106,19 @@ static void writeMvndProperties(Path mvndPropsPath, Path mvndHome, Path javaHome
     static String readTemplate() {
         try (InputStream in = Installer.class.getResourceAsStream("mvnd.properties.template");
                 ByteArrayOutputStream out = new ByteArrayOutputStream(256)) {
-            copy(in, out);
+            copy(in, out, false);
             return new String(out.toByteArray(), StandardCharsets.UTF_8);
         } catch (IOException e) {
             throw new RuntimeException("Could not read build.properties");
         }
     }
 
     static void unzip(Path localZip, Path destinationDir) {
-        LOG.debug("Unzipping {} to {}", localZip, destinationDir);
+        if (LOG.isInfoEnabled()) {
+            LOG.info("Unpacking to {}", destinationDir);
```

Comment:
If `LOG.isInfoEnabled()` is false, it will never log to the DEBUG level.  Should the test be inverted somehow ?

Reply:
Good catch, let me change it somehow

Reply:
Fixed in 832e1ecc150949de5348b0550179e0d04d5df7b0